reo/src/pvwatts.py

```diff
@@ -131,38 +131,34 @@ def __init__(self,
         self.response = None
         self.response = self.data  # store response so don't hit API multiple times
         if self.tilt is None:
-            if self.latitude < 0: # if the site is in the southern hemisphere, and no tilt has been specified, then set the tilt to the positive latitude value and change the azimuth to zero
+            # if the site is in the southern hemisphere, and no tilt has been specified,
+            # then set the tilt to the positive latitude value and change the azimuth to zero
+            if self.latitude < 0:
                 self.tilt = self.latitude * -1
                 self.azimuth = 0
             else:
-                self.tilt = self.latitude # if the site is in the norther hemisphere, and no tilt has been specified, then set the tilt to the latitude value and leave the azimuth at 180
+                # if the site is in the norther hemisphere, and no tilt has been specified,
+                # then set the tilt to the latitude value
+                self.tilt = self.latitude
 
     @property
     def url(self):
         url = self.url_base + "?api_key=" + self.key + "&azimuth=" + str(self.azimuth) + \
               "&system_capacity=" + str(self.system_capacity) + "&losses=" + str(self.losses*100) + \
               "&array_type=" + str(self.array_type) + "&module_type=" + str(self.module_type) + \
-              "&timeframe=" + self.timeframe +"&gcr=" + str(self.gcr) +  "&dc_ac_ratio=" + str(self.dc_ac_ratio) + \
+              "&timeframe=" + self.timeframe + "&gcr=" + str(self.gcr) +  "&dc_ac_ratio=" + str(self.dc_ac_ratio) + \
               "&inv_eff=" + str(self.inv_eff*100) + "&radius=" + str(int(self.radius)) + "&dataset=" + self.dataset + \
               "&lat=" + str(self.latitude) + "&lon=" + str(self.longitude) + "&tilt=" + str(self.tilt)
         return url
 
     @property
     def data(self):
         if self.response is None:
+            # Check if point is beyond the bounds of the NRSDB dataset, if so use the international dataset
+            if self.latitude < -59.5 or self.latitude > 60.01 or self.longitude > -22.37 or self.longitude < -179.58 :
+                self.dataset = 'intl'
+                self.radius = self.radius *2
             resp = requests.get(self.url, verify=self.verify)
-
-            if not resp.ok:
-                data = check_pvwatts_response_data(resp)
-                # if not ok b/c outside of the US, try again with larger search radius in "intl" dataset
-                if ("This location appears to be outside the US" in s for s in data.get("warnings", [])):
-                    self.dataset = "intl"
-                    self.radius = self.radius * 2  # bump up search radius, since there aren't many sites
```

Comment:
would be nice to keep this comment because I did not know why the radius was being doubled until I read it

Reply:
updated the comment